### DISCUSSION TITLE
Checking out latest prototype

### DISCUSSION THREAD

**Discussion by benjamin-cash** [ufs-community/ufs-weather-model#1342] [Q&A] [answered]
What do I need for the latest coupled prototype? Is it just develop at this point?

**Comment by benjamin-cash** [comment, marked as answer]
Answered on slack. 
git clone https://github.com/ufs-community/ufs-weather-model
cd ufs-weather-model
git checkout Prototype-P8
git submodule update --init --recursive

**Reply by JessicaMeixner-NOAA** [reply]
@benjamin-cash we're now on P8 with tag: Prototype-P8

**Reply by benjamin-cash** [reply]
Gotcha, thanks.
